config/katello.migrations/171030135054-foreman-proxy-update.rb

```diff
@@ -0,0 +1,4 @@
+# foreman_proxy defaults for templates
+
+answers['foreman_proxy']['http'] = true if answers['foreman_proxy'].key?('http')
```

Comment:
I think this can simply be ```answers['foreman_proxy']['http'] = true if answers['foreman_proxy']```. In theory ```answers['foreman_proxy']``` can be false or the key could not exist which would make the migration fail/not achieve what you want. It's a little more robust and we should prepare for the unexpected. The same applies below.